<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>1361</width>
     <height>741</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QLabel {
background-color: white;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lvl_one">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>251</width>
     <height>731</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
color: white;
background-color: black;
}

.QPushButton:hover {
background-color: white;
color: black;
}</string>
   </property>
   <property name="text">
    <string>LEVEL 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lvl_two">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>251</width>
     <height>731</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
color: white;
background-color: black;
}

.QPushButton:hover {
background-color: white;
color: black;
}</string>
   </property>
   <property name="text">
    <string>LEVEL 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lvl_three">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>0</y>
     <width>251</width>
     <height>731</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
color: white;
background-color: black;
}

.QPushButton:hover {
background-color: white;
color: black;
}</string>
   </property>
   <property name="text">
    <string>LEVEL 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lvl_four">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>0</y>
     <width>251</width>
     <height>731</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
color: white;
background-color: black;
}

.QPushButton:hover {
background-color: white;
color: black;
}</string>
   </property>
   <property name="text">
    <string>LEVEL 4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lvl_five">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>0</y>
     <width>251</width>
     <height>731</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
color: white;
background-color: black;
}

.QPushButton:hover {
background-color: white;
color: black;
}</string>
   </property>
   <property name="text">
    <string>LEVEL 5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>0</y>
     <width>131</width>
     <height>731</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton {
color: white;
background-color: black;
}

.QPushButton:hover {
background-color: white;
color: black;
}</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
